src/main/java/ca/corefacility/bioinformatics/irida/ria/web/models/TableResponse.java

```diff
@@ -0,0 +1,24 @@
+package ca.corefacility.bioinformatics.irida.ria.web.models;
+
+import java.util.List;
+
+/**
+ * Response sent when items are requested for a table.
+ */
+public class TableResponse {
+	private List<TableModel> dataSource;
+	private Long total;
+
+	public TableResponse(List<TableModel> analyses, Long total) {
```

Comment:
Since the responses for tables has been generalized we should probably change `analyses` here to `dataSource`

Reply:
Thanks, renamed in 10a490cebdb66f6a8be898a4c352097e5d81ab1c